samples/bluetooth/iso_broadcast_benchmark/README.rst

```diff
@@ -0,0 +1,111 @@
+.. _iso_broadcast_benchmark:
+
+Bluetooth: Throughput
+#####################
+
+The ISO Broadcast Benchmark sample tests the quality of a ISO broadcaster
+against one or more ISO broadcast receivers, in terms for packet and sync loss.
+
+
+Overview
+********
+
+The sample transmits data from the *broadcaster* to the
+*receiver* (multiple receivers may be used with no effect on the test)
+and measures the quality of the sync.
+
+The application is used as both a broadcaster and a receiver, and the mode
+can easily be changed.
+
+Requirements
+************
+
+* A board with Bluetooth Low Energy 5.2 and Broadcast Isochronous Channel
+  support
+
+Building and running
+********************
+
+This sample can be found under
+:zephyr_file:`samples/bluetooth/iso_broadcast_benchmark` in the Zephyr tree.
+
+See :ref:`bluetooth samples section <bluetooth-samples>` for details.
+
+
+Testing
+=======
+
+After programming the sample to both boards, test it by performing the following
+steps:
+
+1. Connect to both boards with a terminal emulator (for example, PuTTY or
+   minicom).
+#. Reset both boards.
+#. In one of the terminal emulators, type "r" to start the application on the
+   connected board in the receiver role.
+#. In the other terminal emulator, type "b" to start the application in the
+   broadcaster role.
+#. Optionally modify the broadcasting settings.
+#. Observe that the receiver synchronizes to the broadcaster.
+#. Observe the receive statistics on the receiver(s).
+
+Sample output
+==============
+The receiver will output overall (since boot), current sync (since the BIG was
+synced) and latest 1000 received packets::
```

Comment:
Made it clear that it output statistics. 

> Question: How does this work? Will the statistics also report lost packets (by keeping track of where there should have been packets)?

Yes, both lost and received. Basically the format is `<received>/<expected> - Total packets lost <lost>`

> Are the statistics for the last thousand received packets, or for the last thousand sent packets?

Last thousand received packets I suppose - It's the last 1000 packets we receive from the controller, which may or may not have an SDU from the broadcaster.

Reply:
I think the real questions was whether, in a broadcast, the receiver will always be able to tell that it has lost a packet.
If the packets are supposed to come at fixed times (which I think they are), I guess it will.

Reply:
I am no expert in terms of ISOAL, but whether a packet is lost in a CIS or BIS should give the same result.